### PR TITLE
doc: Coding_convention example changed

### PR DESCRIPTION
**Pull request by eduazocar**
### Contribution description
 In this PR will be provide some examples in the file CODING_CONVENTIONS, this will give to the user an important structure about how to should to write the doxygen documentation headers

### CODE REVIEW

File: CODING_CONVENTIONS.md [hunk 1]
```diff
@@ -259,23 +259,23 @@ An examplary doxygen documentation in a header file can look like this.
 
 ```
 /*
- * Copyright (C) 2014 Peter Schmerzl <peter@schmerzl-os.org>
+ * Copyright (C) 2014 your_name yourlastname <youremailadrr@mail.com>
  *
  * This file is subject to the terms and conditions of the GNU Lesser General
  * Public License v2.1. See the file LICENSE in the top level directory for more
  * details.
  */
```
Comment (luisan00) [inline CODING_CONVENTIONS.md:267]: we are not using this license header

**Review by luisan00 — CHANGES_REQUESTED** [review]
review the proposed changes please

File: CODING_CONVENTIONS.md (hunk 1 above)
Comment (eduazocar) [reply CODING_CONVENTIONS.md:267]: I should use this?
*/ Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0

Comment (luisan00) [reply CODING_CONVENTIONS.md:267]: Yep, I think it's better, in the same line as the proyect license,.
also may be a good Idea to put a reference for the original repo in the footer of this document, what do you think about it?

**Comment by eduazocar** [comment]
I've added the following updates:
update#1 with @luisan00 : we changed the authors indentifier to example
update#2:  changes with the copyrigth model
update#3: changes with the license and added a footer linked with this repo

**Review by luisan00 — APPROVED** [review]

**Review by luisan00 — APPROVED** [review]

**Pull request merged**